x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -20,7 +20,7 @@ export const DOWNLOAD_SOURCE_API_ROOT = `${API_ROOT}/agent_download_sources`;
 export const LIMITED_CONCURRENCY_ROUTE_TAG = 'ingest:limited-concurrency';
 
 // EPM API routes
-const EPM_PACKAGES_MANY = `${EPM_API_ROOT}/packages`;
+export const EPM_PACKAGES_MANY = `${EPM_API_ROOT}/packages`;
```

Comment:
why do you need exporting this constant? i couldn't find it used anywhere in the PR, only exported.

We already export routes that use this constant (see [here](https://github.com/elastic/kibana/blob/b7af143114d9055934f1c7e30f4706ef5b34aec8/x-pack/plugins/fleet/common/constants/routes.ts#L28-L46)), so you might be able to use one of them depending on what you need it for.